code/game/machinery/seed_extractor.dm

```diff
@@ -28,6 +29,29 @@
 
 obj/machinery/seed_extractor/attackby(var/obj/item/O as obj, var/mob/user as mob)
 
+	// Emptying a plant bag
+	if (istype(O,/obj/item/weapon/storage/bag/plants))
+		var/obj/item/weapon/storage/P = O
+		var/loaded = 0
+		for(var/obj/item/seeds/G in P.contents)
+			if(contents.len >= 999)

```

Comment:
apparently some BYOND stuff breaks if it holds 1000 or more items, smartfridge.dm explains in more detail:
var/global/max_n_of_items = 999 // Sorry but the BYOND infinite loop detector doesn't look things over 1000.